src/components/sbb-slider/sbb-slider.scss

```
@@ -0,0 +1,139 @@
+  --sbb-slider-value-fraction: 0;
+  --sbb-slider-knob-size: #{px-to-rem-build(28)};
+  --sbb-slider-knob-border-size: #{px-to-rem-build(3)};
+  --sbb-slider-knob-color: var(--sbb-color-white-default);
+  --sbb-slider-knob-border-color: var(--sbb-color-red-default);
+  --sbb-slider-icon-color: var(--sbb-color-black-default);
+  --sbb-slider-selected-line-color: var(--sbb-color-red-default);
+  --sbb-slider-selected-line-disabled-color: var(--sbb-color-smoke-default);
+  --sbb-slider-line-color: var(--sbb-color-smoke-default);
+  --sbb-slider-line-disabled-color: var(--sbb-color-graphite-default);
+
+  @include ifForcedColors {
+    --sbb-slider-selected-line-color: Highlight;
+    --sbb-slider-line-color: CanvasText;
+  }
+
+  display: inline;
+}
+
+:host([data-sbb-form-field]) {
+  --form-field-overflow: visible;
+  --form-field-background-color: transparent;
+}
+
+.sbb-slider__wrapper {
+  max-width: #{px-to-rem-build(400)};
+  min-width: #{px-to-rem-build(150)};
+  width: 100%;
+  display: flex;
+  gap: var(--sbb-spacing-fixed-2x);
+  padding-block: var(--sbb-spacing-fixed-1x);
+  color: var(--sbb-slider-icon-color);
+}
+
+.sbb-slider__container {
+  position: relative;
+  min-height: var(--sbb-size-icon-ui-small);
+  flex-grow: 1;
+}
+
+.sbb-slider__range-input {
+  position: absolute;
+  width: 100%;
+  inset: 0;
+  opacity: 0;
+  margin: 0;
+}
+
+.sbb-slider__line {
+  @include absoluteCenterY;
+
+  height: var(--sbb-slider-line-height);
+  width: 100%;
+  background-color: var(--sbb-slider-line-color);
+  border-radius: var(--sbb-slider-line-height);
+  overflow: hidden;
+  pointer-events: none;
+}
+
+.sbb-slider__selected-line {
+  position: relative;
+  height: 100%;
+  background-color: var(--sbb-slider-selected-line-color);
+  width: calc(var(--sbb-slider-value-fraction) * 100%);
+}
+
+.sbb-slider__knob {
+  @include absoluteCenterY;
+  @include shadow-level-5-hard;
+
+  width: var(--sbb-slider-knob-size);
+  height: var(--sbb-slider-knob-size);
+  inset-inline-start: calc(
+    calc(var(--sbb-slider-value-fraction) * 100%) - var(--sbb-slider-knob-size) *
+      var(--sbb-slider-value-fraction)
+  );
+  transition-property: width, height;
+  transition-duration: var(--sbb-animation-duration-1x);
+  transition-timing-function: ease-in;
+  background-color: var(--sbb-slider-knob-color);
+  border: var(--sbb-slider-knob-border-size) solid var(--sbb-slider-knob-border-color);
+  border-radius: 50%;
+  pointer-events: none;
+}
+
+// override icons color
+:host([disabled]) {
+  --sbb-slider-icon-color: var(--sbb-color-graphite-default);
+
+  .sbb-slider__knob {
+    --sbb-slider-knob-color: var(--sbb-color-milk-default);
+    --sbb-slider-knob-border-color: var(--sbb-slider-line-disabled-color);
+  }
+}
+
+:host([readonly]) {
+  --sbb-slider-icon-color: var(--sbb-color-smoke-default);
+
+  .sbb-slider__knob {
+    --sbb-slider-knob-border-color: var(--sbb-slider-selected-line-disabled-color);
+  }
```

selector should not be necessary here
```suggestion
    --sbb-slider-knob-border-color: var(--sbb-slider-selected-line-disabled-color);

```